integration/test_flow_integration.py

```diff
@@ -462,3 +462,27 @@ def test_write_multiple_keys_to_v3io_from_csv(setup_teardown_test):
     assert response.status_code == 200
     assert expected == response.output.item
 
+
+def test_write_none_time(setup_teardown_test):
```

Comment:
Shouldn't we try to read it back too?

Reply:
i am reading it in the end of the test (get kv item)